### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Curious-Yu**
Added Reddit Topic

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our contribution guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
@Curious-Yu have you tested this out with blog post workflow?. You cant give random URL to it, it should be something that returns the rss feed.

**Comment by gautamkrishnar** [comment]
Please reopen the PR if you are able to find the RSS feed URL of the topic.

**Comment by gautamkrishnar** [comment]
@Curious-Yu Just did some research on this: https://www.reddit.com/wiki/rss

File: README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ Following are the list of some popular blogging platforms and their RSS feed url
 | [Anchor.fm Podcasts](https://anchor.fm/) | `https://anchor.fm/s/podcastId/podcast/rss` | You can get the rss feed url of a podcast by following [these](https://help.anchor.fm/hc/en-us/articles/360027712351-Locating-your-Anchor-RSS-feed) instructions | https://anchor.fm/s/1e784a38/podcast/rss |
 | [Hashnode](https://hashnode.com/) | `https://@username.hashnode.dev/rss.xml` | Replace @username with your Hashnode username | https://polilluminato.hashnode.dev/rss.xml |
 | [Google Podcasts](https://podcasts.google.com/) | `https://podcasts.google.com/feed/channelId` | Replace `channelId` with your Google podcast channel Id | https://podcasts.google.com/feed/aHR0cHM6Ly9mZWVkcy5zb3VuZGNsb3VkLmNvbS91c2Vycy9zb3VuZGNsb3VkOnVzZXJzOjYyOTIxMTkwL3NvdW5kcy5yc3M= |
+| [Reddit](https://www.reddit.com/) | `https://www.reddit.com/r/topic/` | Replace `topic` (example: r/gadgets or r/bayarea) with Reddit topic that interest you or localized to you.| https://www.reddit.com/r/bayarea/ |
```
Comment (gautamkrishnar) [inline README.md:110]: Feed URL: `https://any-reddit.com-url.rss`
Comments: `You can create an RSS feed by adding ".rss" to the end of an existing Reddit URL`
Example: `https://www.reddit.com/r/bayarea.rss`

Please make these changes

**Review by gautamkrishnar — CHANGES_REQUESTED** [review]
Please update with the changes.

**Comment by Curious-Yu** [comment]
> Please update with the changes.

Yes. sorry about the missing part. now should be perfect. Thanks.

**Review by gautamkrishnar — APPROVED** [review]
LGTM 👍

**Comment by gautamkrishnar** [comment]
@Curious-Yu Thanks a lot for contributing.

**Pull request merged**